test/suite/wtbackup.py

```diff
@@ -25,33 +25,38 @@
 # OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
 # ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
 # OTHER DEALINGS IN THE SOFTWARE.
-import os, glob
+import os, glob, shutil
 import wttest, wiredtiger
 from suite_subprocess import suite_subprocess
 from helper import compare_files
 
 # Shared base class used by backup tests.
 class backup_base(wttest.WiredTigerTestCase, suite_subprocess):
-    cursor_config = None        # a config string for cursors
+    data_cursor_config = None    # a config string for cursors
     mult = 0                    # counter to have variance in data
     nops = 100                  # number of operations added to uri
 
-    # We use counter to produce unique backup names for multiple iterations
-    # of incremental backup tests.
-    counter = 0
+    # We use counter to produce unique backup ids for multiple iterations
+    # of incremental backup.
+    bkup_id = 0
     # To determine whether to increase/decrease counter, which determines
-    initial_backup = True
+    initial_backup = False
     # Used for populate function
     rows = 100
     populate_big = None
 
+    # Specify a logpath directory to be used to place wiredtiger log files
```

Comment:
Here and other places, all comments should be full sentences, ending in a period.

Reply:
done!